src/main/java/Ui.java

```diff
@@ -0,0 +1,185 @@
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Ui {
+
+    Scanner sc;
+
+    public Ui() {
+        sc = new Scanner(System.in);
+    }
+
+    public void printGreeting() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        String greetMsg = "    ____________________________________________________________\n" +
+                "     Hello! I'm Duke\n" +
+                "     What can I do for you?\n" +
+                "    ____________________________________________________________\n";
+        System.out.println(logo + '\n' + greetMsg);
+    }
+
+    public String getUserInput() {
+        String userInput = sc.nextLine();
+        return userInput;
+    }
+
+    public static void handleBye() {
+        System.out.println("    ____________________________________________________________\n" +
```

Comment:
The '___________' dividers can probably be included in a showLine() method like I have done in my code, to make everything neater and not exceed the amount of characters in a specific line

Reply:
oh yeah i alr did that but seems like my PR did not update :/